### DISCUSSION TITLE
blog/2023-03-28-causal-inference-with-gamma-regression-or-the-problem-is-the-link-function-not-the-likelihood/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#16] [Comments]
# blog/2023-03-28-causal-inference-with-gamma-regression-or-the-problem-is-the-link-function-not-the-likelihood/

Part 6 of the GLM and causal inference series.

https://timely-flan-2986f4.netlify.app/blog/2023-03-28-causal-inference-with-gamma-regression-or-the-problem-is-the-link-function-not-the-likelihood/

**Comment by sjwild** [comment]
A few comments:
* You mention the there is an inflection point at 1. I don't think there is, because inflection points are where the curvature changes sign (e.g., from positive to negative). It looks like the derivative is negative and stays that way, albeit becoming less negative as x increases. (I might be wrong here. It's been a long time since I did anything calculus).
* Pretty sure you can interpret the $\beta$ coefficient for prelc as an elasticity in the gaussian model with a log link (I know you aren't interpreting it, so what follows is an aside). The difference between gaussian with log link vs lognormal models are in the error (IIRC), with lognormal models having error on the log scale. In the gaussian(link - "log") model, I'm pretty sure it works out to 

$$y \sim Normal(e^{X\beta}, \sigma)$$

* If that is the case, you can interpret $\beta$ as an elasticity if X is logged. So a 1% increase in X leads to a $\beta$% increase in Y. I believe this holds for the Gamma(link = "log") model as well, though I need to double check tomorrow (it holds for poisson, so I don't see why not. Naturally, I am probably wrong about Gamma and elasticities).
* I could not for the life of me interpret coefficients in a Gamma regression.
* Good post overall.

**Reply by ASKurz** [reply]
@sjwild, if what happens at 1 isn't properly described as an inflection point, how would you describe it? Is it an elbow?

As to the elasticity points, that's all well outside of my competence. I don't think I've even heard another psychologist use the language of elasticity.

**Reply by sjwild** [reply]
I don't know. "Elbow" sounds right to me.

Elasticities tens to be more of an econometrics thing. I don't think I've ever seen them used in any psychology paper I've read.
